.github/workflows/ci.yml

```diff
@@ -427,3 +427,13 @@ jobs:
       uses: pypa/gh-action-pypi-publish@release/v1
       with:
         skip-existing: true # prevent against pushing duplicate versions
+
+  notify_on_release:
+    needs: [build-n-publish]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Announce Release
+        id: slack
+        uses: slackapi/slack-github-action@v1.24.0
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_RELEASE_WEBHOOK_URL }}
```

Comment:
Do we need a payload here?  Just comparing it to the failure Slack notification PR

Reply:
Good question! We don't need a payload here because the message is static. It will look like this:

<img width="587" alt="Screenshot 2023-08-04 at 12 53 02 PM" src="https://github.com/great-expectations/great_expectations/assets/12725591/871af4d4-1916-4e63-98c7-6e3d3ab32e1d">

The failure notification had a dynamically generated link. The docs describing this are here:
https://github.com/slackapi/slack-github-action#usage